package.json

```diff
@@ -27,11 +27,11 @@
     ],
     "license": "MIT",
     "dependencies": {
+        "ddq": "2.0.0",
```

Comment:
Fixed that. The build will fail though until DDQ is published. Tests are passing locally. My commits 
46eee4607d09d2f88ca247764e170ef043da943b
fb9ac39046289a652a13be072a4d0939a129ed38